### ISSUE TITLE
nf-core modules update changed md5sum of skipped module in modules.json

### ISSUE THREAD

**Issue by FriederikeHanssen** [nf-core/tools#2014]
### Description of the bug

Hey!

I ran `nf-core modules update -a` and then selected the second option (show diff and y/N before applying it). For two modules the md5sum was changed, even though I selected N and skipped the update:

see here:

https://github.com/nf-core/sarek/pull/822/files#diff-97c7fe3fe8f628d2435d00b93af4cb47052e8e1f3d33b7334d1a031fd15415d7

### Command used and terminal output

```console
nf-core modules update -a
```

### System information

Using nf-core/tools 2.6

**Comment by mirpedrol** [comment]
Fixed by #2017

**Comment by FriederikeHanssen** [comment]
Hey! Sorry to bring this up again, but it just happened again. I am on 2.8. I ran

```
nf-core/modules update

? Update all modules or a single named module? All modules
? Do you want to view diffs of the proposed changes? Preview diff in terminal, choose whether to update files

? Update module 'controlfreec/makegraph'? No

Yet the markgraph checksum was updated.

**Comment by mirpedrol** [comment]
Now yes, fixed by #2323 😄
